dexbot/strategies/external_feeds/ccxt_feed.py

```diff
@@ -128,11 +111,10 @@ def test_feed(exchange, symbol):
                     'bid: ' + str(ticker['bid']),
                     'ask: ' + str(ticker['ask']),
                     'volume: ' + str(ticker['quoteVolume']))
-            else: 
+            else:
                 print_args('Symbol not found')
-                print_exch_symbols(exch)
+                print_exchange_symbols(exch)
```

Comment:
IDE does not recognize where `print_exchange_symbols` comes from. Missing import?